### ISSUE TITLE
Textual definition update: GO:1990244 histone H2AT120 kinase activity

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#24715]
* **GO term ID and label for which you request a definition update**
GO:1990244 histone H2AT120 kinase activity

* **New proposed definition (genus-differentia pattern, plus additional information if needed)**

Current def is "Catalysis of the transfer of a phosphate group to the threonine-120 residue of histone H2A. PMID:24140421".  But this doesn't specify in which species/group the threonine is at residue 120.  E.g. in Drosophila, T120 is corresponds to T119 (PMID: 15078818).
Looks like the paper used for the original definition used human cell lines, so maybe the new def should be something like "Catalysis of the transfer of a phosphate group to the threonine-120 residue of histone H2A (residue numbering refers to human H2A). PMID:24140421"

* **Reference(s)**
PMID: [15078818](https://pubmed.ncbi.nlm.nih.gov/15078818)

* **Additional information**
* Might be other similar cases where the stated residue number isn't universally correct....

**Comment by pgaudet** [comment]
We dont systematically have this - I suppose we should do this for every histone modification term ?

**Comment by sjm41** [comment]
Hmmm, I guess the first thing to do is to figure out how big a problem this is.  Seems that histone H2A varies slightly between flies and humans, but I haven't checked the other histones, or compared more distant species....  Maybe all other key residues are at exactly the same position in other histones/species (though I doubt it)??

Assuming that any given numbering in a GO term name/definition is at least correct for humans (or mammals/vertebrates), then adding a caveat/note along the lines of "residue numbering according to human" to every histone modification term mentioning a specific residue would at least address this point.  (eg., for the GO:1990244 example, I almost made a ticket to request a new term for 'histone H2AT119 kinase activity' before I checked that human T120 = fly T119)

**Comment by pgaudet** [comment]
There seem to ba many positions that vary on most of the histones, see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5732057/figure/F1/

**Comment by deustp01** [comment]
> Hmmm, I guess the first thing to do is to figure out how big a problem this is.

A fix that can be generalized would be good here. I don't know of any handy statistics, but it's common to observe that an amino acid sequence motif is well conserved across species, and the modification of a very well conserved amino acid residue within that motif has well-conserved functional consequences, BUT the exact numbering of the residues of the motif differs between species because of indels accumulated elsewhere in the protein over evolution.

Is there general agreement that all annotation of this sort shall be with reference to a human canonical protein and its UniProt amino acid sequence? And is the UniProt human proteome a generally accepted starting point for this?

**Comment by sjm41** [comment]
_Is there general agreement that all annotation of this sort shall be with reference to a human canonical protein and its UniProt amino acid sequence? And is the UniProt human proteome a generally accepted starting point for this?_

That seems like the only practical solution here, and I assume it's already the case but just isn't explicitly stated in the definitions/comments.

**Comment by pgaudet** [comment]
Any preference as to whether this should go in the comment or in the definition? It seems more appropriate for a comment, the only concern would be that this is not sufficiently visible. What do you think ?

**Comment by sjm41** [comment]
Yes, it does seem more appropriate as a comment.  And then you can just add the same generic comment to all affected terms.

**Comment by sjm41** [comment]
Hi @pgaudet 

This issue came up briefly at the recent Geneva GOC meeting, and I think you said we had a policy of using the human coordinates, with a note saying as much included in the GO term def/comment line. Is that right?

This entry, at least, still needs a note adding to say that the human coordinates are being used:
(in Drosophila, T120 is corresponds to T119 (PMID: 15078818).)

id: GO:1990244
name: histone H2AT120 kinase activity
namespace: molecular_function
def: "Catalysis of the reaction: histone H2A-threonine (position 120) + ATP = histone H2A-phosphothreonine (position 120) + ADP. This reaction is the addition of a phosphate group to the threonine residue at position 120 of histone H2A." [PMID:24140421]
synonym: "histone H2A-T120 kinase activity" EXACT []
synonym: "histone kinase activity (H2A-T120 specific)" EXACT []
synonym: "histone threonine kinase activity (H2A-T120 specific)" EXACT []
xref: EC:2.7.11.1 {source="skos:broadMatch"}
is_a: GO:0004674 ! protein serine/threonine kinase activity
is_a: GO:0140995 ! histone H2A kinase activity
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
created_by: sp
creation_date: 2013-12-04T06:38:32Z

What should we do here?

**Comment by pgaudet** [comment]
Hi Steven, 

I have made some progress here: 
https://docs.google.com/spreadsheets/d/1WrRx8X6f4QyBwS93SD6_YL-jfwsA9BpAdTPzY1WXgkQ/edit?gid=368119721#gid=368119721
coming from this alignment: https://docs.google.com/document/d/1gRooXD7vgz4RcwL8HCZVs6bb6Lr93lhTi0bLWNFL0aA/edit?tab=t.0#

However I ned to rediscuss this with @colinlog  tomorrow - I am not 100% sure that we have the right sequence for H2A. 

Meanwhile your feedback would be more than welcome on what is proposed on the spreadsheet. 

Thanks, Pascale

**Comment by sjm41** [comment]
Thanks - looks good!
I added a comment about H2A T119/T120 in https://docs.google.com/spreadsheets/d/1WrRx8X6f4QyBwS93SD6_YL-jfwsA9BpAdTPzY1WXgkQ/edit?gid=368119721#gid=368119721

Is there a general ticket about adding these comments linked to your Google documents? if so, I'll close this specific ticket.

**Comment by pgaudet** [comment]
No ticket yet, I was waiting to finish that proposal. Hopefully tomorrow!

**Comment by sjm41** [comment]
This is done, closing - thanks @pgaudet !


id: GO:1990244
name: histone H2AT120 kinase activity
namespace: molecular_function
def: "Catalysis of the reaction: histone H2A-threonine (position 120) + ATP = histone H2A-phosphothreonine (position 120) + ADP. This reaction is the addition of a phosphate group to the threonine residue at position 120 of histone H2A." [PMID:24140421]
comment: Note that the residue position corresponds to the canonical human H2A2A histone (UniProtKB:Q6FI13); this residue is conserved across all eukaryotes, but corresponds to T119 in Drosophila. Residue 1 is the first residue following removal of the initiating Methionine (Met). Note that each histone is encoded by multiple genes, and sequences may vary across different genes within an organism.
synonym: "histone H2A-T120 kinase activity" EXACT []
synonym: "histone kinase activity (H2A-T120 specific)" EXACT []
synonym: "histone threonine kinase activity (H2A-T120 specific)" EXACT []
xref: EC:2.7.11.1 {source="skos:broadMatch"}
is_a: GO:0004674 ! protein serine/threonine kinase activity
is_a: GO:0140995 ! histone H2A kinase activity
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30578" xsd:anyURI
created_by: sp
creation_date: 2013-12-04T06:38:32Z
